renderer.php

```diff
@@ -2,7 +2,7 @@
 // must be run within Dokuwiki
 if(!defined('DOKU_INC')) die();
 
-require_once DOKU_INC.'inc/parser/renderer.php';
+require_once DOKU_INC.'inc/Parsing/renderer.php';
 require_once DOKU_INC.'inc/fulltext.php';
```

Comment:
The proper fix would be to remove these lines completely. DokuWiki's autoloading should take care of loading the files when needed.